pkg/cmd/cli/describe/deployments.go

```diff
@@ -224,6 +207,34 @@ func printHook(prefix string, hook *deployapi.LifecycleHook, w io.Writer) {
 	}
 }
 
+func printAutoscalingInfo(config *deployapi.DeploymentConfig, kclient kclient.Interface, w *tabwriter.Writer) {
```

Comment:
Seems like you want a `GroupResource` and a `name`, right?  That would make this re-usable instead of having to create separate ones for each type.